src/vue/backend/src/views/GuidedSetupConnectIDXBrokerAccount.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+    <GuidedSetupContentCard class="gs" cardTitle="Connect Your IDX Broker Account" :steps="steps" :relatedLinks="links">
+        <template v-slot:description>
+            <p>This step is optional. A sentence or two about why you should connect IMPress for IDX Broker to your IDX Broker account. Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+        </template>
+        <template v-slot:controls>
+            <APIKey/>
```

Comment:
This needs to take in the success, error, and loading states. Those are determined by an api call done at this level. Since we dont not have these connections yet, you can do a mock action with a time out and. `to-do` that way it is all incorporated later.

Reply:
I'm not sure how to achieve the loading state, but I set up placeholders for the success and error conditions